src/plugins/intel_cpu/tests/unit/CMakeLists.txt

```diff
@@ -60,6 +60,7 @@ ov_add_test_target(
             ov_models
             ov_snippets_models
             snippets_test_utils
+            sharedTestClasses
```

Comment:
Why? `sharedtestsclasses` can be shared between functional tests because they use API level testing